romanesco/executors/__init__.py

```diff
@@ -1 +1 @@
-from . import docker, python, r, workflow
+from . import docker, python, r, workflow, pyspark
```

Comment:
I am happy to add some basic plugin infrastructure whenever we are ready to pull the trigger on that.